lib/core/src/client/preview/start.js

```diff
@@ -226,8 +226,10 @@ export default function start(render, { decorateStory } = {}) {
       case 'story':
       default: {
         if (getDecorated) {
-          render(renderContext);
-          addons.getChannel().emit(Events.STORY_RENDERED, id);
+          (async () => {
```

Comment:
Do we need to catch here?

Reply:
What should be the handler? `logger.error`?

Reply:
👍

Reply:
I can't find a way to pass error callback to `ReactDOM.render` which is the only source of asynchrony here.

Looks like they just use `then` without `catch`: https://github.com/facebook/react/blob/master/packages/react-dom/src/client/ReactDOM.js#L393
So we won't actually catch anything. Hopefully, that means that React does the error handling for us